README.md

```diff
@@ -27,37 +27,40 @@ See the [list of releases](../../releases).
 * For those willing to help testing or developing the latest features, [the dev branch](https://github.com/FreshRSS/FreshRSS/tree/dev) is waiting for you!
 
 # Disclaimer
-This application was developed to fulfil personal needs primarily, and comes with absolutely no warranty.
-Feature requests, bug reports, and other contributions are welcome. The best way is to [open an issue on GitHub](https://github.com/FreshRSS/FreshRSS/issues).
+This application was primarily developed to fulfill personal needs, and comes with absolutely no warranty.
```

Comment:
If we keep this in, I'd probably make it `originally`.

Reply:
I'd agree with that, but I also think that everything before the coma could be cut at this point